docs/man/qdstat.8.adoc

```diff
@@ -412,6 +412,285 @@ The operational status of this auto link:
+for addresses that have fallback capability enabled.
+
+Tables for each vhost and user group.
+
+Vhost::
+Name of vhost.
+
+UserGroup::
+Name of vhost usergroup.
+
+vhost::
+Name of vhost.
+
+group::
+Name of vhost usergroup.
+
+users::
+A list of authenticated users for this user group.
+
+remoteHosts::
+A list of remote hosts from which the users may connect. A host can
+be a hostname, IP address, or IP address range. Use commas to separate
+multiple hosts. To allow access from all remote hosts, specify a
+wildcard 'asterisk \*'. To deny access from all remote hosts, leave this
+attribute blank.
+
+sources::
+A list of source addresses from which users in this group may receive messages.
+To specify multiple addresses, separate the addresses with either a comma or
+a space. If you do not specify any addresses, users in this group are not
+allowed to receive messages from any addresses. You can use the substitution
+token '${user}' to specify an address that contains a user's authenticated
+user name. You can use an 'asterisk \*' wildcard to match one or more
+characters in an address. However, this wildcard is only recognized if
+it is the last character in the address name. You may specify attributes
+'sources' or 'sourcePattern' but not both at the same time.
+
+targets::
+A list of target addresses to which users in this group may send messages.
+To specify multiple addresses, separate the addresses with either a comma or
+a space. If you do not specify any addresses, users in this group are not
+allowed to send messages to any addresses. You can use the substitution
+token '${user}' to specify an address that contains a user's authenticated
+user name. You can use an 'asterisk \*' wildcard to match one or more
+characters in an address. However, this wildcard is only recognized if it
+is the last character in the address name. You may specify attributes
+'targets' or 'targetPattern' but not both at the same time.
+
+sourcePattern::
+A wildcarded pattern for matching source addresses from which users in
+this group may receive messages. The pattern consists of one or more
+tokens separated by a forward slash '/'. A token can be one of the
+following: 'asterisk \*', 'hash \#', or a sequence of characters
+that do not include '/', 'asterisk \*', or 'hash \#'.  The 'asterisk \*' token matches any single token.
+The 'hash \#' token matches zero or more tokens. 'asterisk \*' has higher precedence than 'hash \#',
+and exact match has the highest precedence. To specify multiple addresses,
+separate the addresses with either a comma or a space. You can use the
+text string '${user}' in a token to specify an address that contains a
+user's authenticated user name. If you do not specify any addresses then
+users in this group are not allowed to receive messages from any addresses.
+You may specify attributes 'sources' or 'sourcePattern' but not both at
+the same time.
+
+targetPattern::
```

Comment:
I think this section is good for the man page, but might need some teasing out in docs? wdyt?